__tests__/topics.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const app = require('../app/app');
+
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+  test('GET /api/topics - success', async () => {
+    const response = await request(app).get('/api/topics');
+    expect(response.status).toBe(200);
+    expect(response.body.topics).toBeInstanceOf(Array);
+    response.body.topics.forEach(topic => {
+      expect(topic).toHaveProperty('slug');
+      expect(topic).toHaveProperty('description');
+    });
```

Comment:
Great test. As it's gonna reseed before each test, so the array won't change, thus, it would be better to check the length of the array as well. And to be more specific what the datatype of the values of slug and description. (`object.containing` would help)